.drone-1.0.yml

```
@@ -33,7 +33,7 @@ steps:
       NPM_AUTH_TOKEN:
         from_secret: npm_auth_token
     commands:
-      - npm audit --audit-level=high --production
+      - npx audit-ci --high --skip-dev --config .auditrc.json
```

Could we alias this into an npm script so it's easier to run locally?

i.e. add this 👇 to the `scripts` in the package json:

```
"test:audit": "audit-ci --high --skip-dev --config .auditrc.json"
```
and also 
```suggestion
      - npm run test:audit
```